livekit-agents/livekit/agents/worker.py

```diff
@@ -690,15 +703,24 @@ def _on_process_job_launched(self, proc: ipc.job_executor.JobExecutor) -> None:
         self._update_job_status_sync(proc)
 
     async def _update_worker_status(self):
+        job_cnt = len(self._proc_pool.get_running_jobs())
         if self._draining:
-            update = agent.UpdateWorkerStatus(status=agent.WorkerStatus.WS_FULL)
+            update = agent.UpdateWorkerStatus(
+                status=agent.WorkerStatus.WS_FULL, job_count=job_cnt
+            )
             msg = agent.WorkerMessage(update_worker=update)
             await self._queue_msg(msg)
             return
 
-        current_load = await asyncio.get_event_loop().run_in_executor(
-            None, self._opts.load_fnc
-        )
+        def load_fnc():
+            signature = inspect.signature(self._opts.load_fnc)
+            parameters = list(signature.parameters.values())
+            if len(parameters) == 0:
+                return self._opts.load_fnc()  # type: ignore
```

Comment:
not sure about the `type: ignore` here
What are they used for?

Reply:
type:ignore is needed because it's not known which load_fnc we have, one with arg or one without?